test/filters-cases/bug-1229.asm.directives.approved.txt

```diff
@@ -57,7 +57,7 @@
       "labels": [],
       "source": {
         "column": 12,
-        "file": "/tmp/compiler-explorer-compiler1181120-2080-yfo6a1.y1o4e//opt/compiler-explorer/libs/ctre/master/include/ctre/return_type.hpp",
+        "file": "/opt/compiler-explorer/libs/ctre/master/include/ctre/return_type.hpp",
```

Comment:
Wowser, that's a lot of changes. thanks! (i'm hoping the "reapprove all the things" helped.